parallel/tutorial.md

```diff
@@ -38,111 +38,121 @@ threads can be used simultaneously to do the computation, which comprises the se
-copy them to the cloned `gazebo` and `sdformat` source folder as `package.xml`
+The effectiveness of the parallelization strategies described in this tutorial depends on the scenario that is being simulated. For example, the threaded islands strategy can parallelize simulation of multiple non-interacting robots, while the position error correction strategy can parallelize simulation of individual robot. The following test scenarios
+vary the number and complexity of simulated entities to show the performance of each parallelization strategy.
 
-~~~
-curl https://bitbucket.org/scpeters/unix-stuff/raw/master/package_xml/package_gazebo.xml > ${WS}/src/gazebo/package.xml
-curl https://bitbucket.org/scpeters/unix-stuff/raw/master/package_xml/package_sdformat.xml > ${WS}/src/sdformat/package.xml
-~~~
+* [revolute\_joint\_test.world](https://github.com/osrf/gazebo/blob/diagnostics_scpeters/test/worlds/revolute_joint_test.world#L12). The `revolute_joint_test.world` file is used in Gazebo’s automated test system. The world file includes eight instances of the double pendulum with base model arrayed in a circle. The model consists of a base in contact with the ground that is connected to two links by revolute joints. The models are arranged in close proximity but do not contact each other. This scenario includes contact, articulation constraints, and multiple *islands*.
+![](files/pendulums.jpg)
 
-Gazebo can then be built using the `catkin_make_isolated` command:
+* [pr2.world](https://github.com/osrf/gazebo/blob/diagnostics_scpeters/worlds/pr2.world#L12): The `pr2.world` file includes a PR2 robot on a flat ground plane. There are no other objects with which to interact. The PR2 is a complex robot with 48 rigid bodies and 58 articulation joints. This scenario includes contact and articulation constraints but with only one *island*.
+![](files/pr2.jpg)
 
-~~~
-cd ${WS}
-./src/catkin/bin/catkin_make_isolated
-~~~
+* [dual_pr2.world](https://github.com/osrf/gazebo/blob/diagnostics_scpeters/worlds/dual_pr2.world#L12). The `dual_pr2.world` file includes two PR2 robots on a flat ground plane. The robots do not interact with each other. This scenario is similar to `pr2.world`, but it includes two *islands*, so that the effect of each parallelization strategy can be compared with complex robots.
+![](files/dual_pr2.jpg)
+
+### Running some experiments
 
-If you want to use diagnostic timers to evaluate performance,
-define the `ENABLE_DIAGNOSTICS` symbol during compilation.
-This will output the diagnostic timing data to the `~/.gazebo/diagnostics` folder.
+For a quick review of how parallelization works in Gazebo we will run some experiments. For a deeper review of the experiments please refer to the [Gazebo parallel physics report](http://gazebosim.org/assets/parallel_physics-1f40fad62e6878895798c9cb3261d92164a083c2fdbdb18a09d0891fafdc5230.pdf).
+
+To run the experiment:
```

Comment:
Maybe it's not clear to the user that they need to be in a directory that has those worlds. How about adding a `wget` command to fetch it before running?